cocos/network/HttpClient.cpp

```diff
@@ -510,93 +510,93 @@ void HttpClient::dispatchResponseCallbacks()
 // Process Response
 void HttpClient::processResponse(HttpResponse* response, char* responseMessage)
 {
-	auto request = response->getHttpRequest();
-	long responseCode = -1;
-	int retValue = 0;
-
-	// Process the request -> get response packet
-	switch (request->getRequestType())
-	{
-	case HttpRequest::Type::GET: // HTTP GET
-		retValue = processGetTask(this, request,
-			writeData,
-			response->getResponseData(),
-			&responseCode,
-			writeHeaderData,
-			response->getResponseHeader(),
-			responseMessage);
-		break;
-
-	case HttpRequest::Type::POST: // HTTP POST
-		retValue = processPostTask(this, request,
-			writeData,
-			response->getResponseData(),
-			&responseCode,
-			writeHeaderData,
-			response->getResponseHeader(),
-			responseMessage);
-		break;
-
-	case HttpRequest::Type::PUT:
-		retValue = processPutTask(this, request,
-			writeData,
-			response->getResponseData(),
-			&responseCode,
-			writeHeaderData,
-			response->getResponseHeader(),
-			responseMessage);
-		break;
-
-	case HttpRequest::Type::DELETE:
-		retValue = processDeleteTask(this, request,
-			writeData,
-			response->getResponseData(),
-			&responseCode,
-			writeHeaderData,
-			response->getResponseHeader(),
-			responseMessage);
-		break;
-
-	default:
-		CCASSERT(false, "CCHttpClient: unknown request type, only GET,POST,PUT or DELETE is supported");
```

Comment:
I didn't notice the modifications here, just applied with the indent, will roll it back